### PR TITLE
Improve Oban integration docs (Elixir)

### PR DESCRIPTION
**Pull request by whatyouhide**
## DESCRIBE YOUR PR

Doc fixes inspired by https://github.com/getsentry/sentry-elixir/issues/757.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/platforms/elixir/integrations/oban/index.mdx [hunk 1]
```diff
@@ -33,3 +32,19 @@ config :sentry,
 ```
 
 This configuration will report failed Oban jobs to Sentry. It will also report started, completed, and failed jobs, as well as their duration. It will use the worker name as the `monitor_slug` of the reported cron.
+
+<Alert level="info" title="Perform Errors">
+Oban wraps *unhandled* crashes, exits, and throws in jobs inside [`Oban.CrashError`](https://hexdocs.pm/oban/2.18.0/Oban.CrashError.html) exceptions. If jobs return errors manually however, such as by returning `{:error, reason}` tuples, then Oban wraps those inside [`Oban.PerformError`](https://hexdocs.pm/oban/2.18.0/Oban.PerformError.html) exceptions. If you want to only report crashes to Sentry, you can configure an event filter that doesn't report `Oban.PerformError` events or that inspects its `:reason` field to determine whether to report them. For example:
```
Comment (lizokm) [inline docs/platforms/elixir/integrations/oban/index.mdx:37]:
```suggestion
<Alert level="info" title="Perform Errors">
Oban wraps *unhandled* crashes, exits, and throws in jobs inside [`Oban.CrashError`](https://hexdocs.pm/oban/2.18.0/Oban.CrashError.html) exceptions. However, if jobs return errors manually (such as by returning `{:error, reason}` tuples) then Oban will wrap those inside [`Oban.PerformError`](https://hexdocs.pm/oban/2.18.0/Oban.PerformError.html) exceptions. If you only want to report crashes to Sentry, you can either configure an event filter that doesn't report `Oban.PerformError` events, or that inspects its `:reason` field to determine whether to report them. For example:
```

**Review by lizokm — APPROVED** [review]
Thanks for contributing to our docs!

**Review by sl0thentr0py — APPROVED** [review]

**Comment by chargome** [comment]
@whatyouhide can you please rebase your pr

**Comment by whatyouhide** [comment]
@chargome done!

**Pull request merged**